docs/README.md

```diff
@@ -48,7 +48,7 @@ class BookRecommendationPrompt(Prompt):
 prompt = BookRecommendationPrompt(topic="coding")
 print(str(prompt))
 
-model = OpenAIChat(api_key=os.getenv("OPENAI_API_KEY"))
+model = OpenAIChat(api_key="YOUR_OPENAI_API_KEY")
```

Comment:
for example here too I'm thinking maybe we want to just set the os environ

Reply:
see above comment